compiler/src/iree/compiler/Codegen/LLVMGPU/Utils/LLVMGPULayoutAnalysisAndDistribution.cpp

```diff
@@ -343,6 +344,35 @@ void propagateLayoutToFor(scf::ForOp forOp,
   }
 }
 
+static void propagateLayoutToOthers(SmallVectorImpl<Value> &operands,
+                                    DenseMap<Value, Layout> &layoutMap) {
+  int numOperands = operands.size();
+  // Find an operand with a layout
+  int i;
+  for (i = 0; i < numOperands; i++) {
+    if (layoutMap.count(operands[i])) break;
+  }
+  // Propagate layout to others
+  for (int j = 0; j < numOperands; j++) {
+    if (j == i) continue;
+    if (!layoutMap.count(operands[j])) {
+      layoutMap.try_emplace(operands[j], layoutMap.at(operands[i]));
+      layoutMap.at(operands[j]).debugPrint("binary/unary operand");
+    } else {
+      assert(layoutMap.at(operands[i]) == layoutMap.at(operands[j]));
+    }
+  }
+}
+
+void propagateLayoutToElementwiseOp(Operation *op,
```

Comment:
nit: make this static (also other functions) for consistency. I know this is in an anonymous namespace, usually the convention is that private functions are mark as static and don't need to be in anonymous namspace.

Reply:
sounds good. I will make them all static.